config/locales/headings/en.yml

```diff
@@ -1,39 +1,39 @@
 en:
   headings:
     confirmations:
-      new: Resend confirmation instructions
+      new: Send another confirmation email
     contact: Contact us
-    create_account_with_sp: Make a login.gov account to continue to %{sp}
-    create_account_without_sp: Make a login.gov account
+    create_account_with_sp: Create a login.gov account to continue to %{sp}
+    create_account_without_sp: Create a login.gov account
     edit_info:
-      email: Edit your email
-      password: Edit your password
-      phone: Edit your phone number
+      email: Change your email
+      password: Change your password
+      phone: Change your phone number
     log_in: Sign in
-    log_in_branded: Sign in to continue to %{sp}
-    choose_otp_delivery: Choose delivery method
+    log_in_branded: Sign in to continue to %{sp} #question about variable names -- "_branded" vs _"w_with_SP"?
+    choose_otp_delivery: Choose how you'd like to receive a one-time passcode
     passwords:
       change: Change your password
-      forgot: Forgot password
-      confirm: Confirm your password
+      forgot: Forget your password? #why is this here and not in links/en.yml?
+      confirm: Confirm your password to continue
     privacy_policy:
       privacy_act: Privacy Act Statement
-      security_practices: Our Security Practices
-      privacy_practices: Our Privacy Practices
+      security_practices: Our Security Practices #should this be in sentence case?
+      privacy_practices: Our Privacy Practices #should this be in sentence case?
       authorities: Authorities
       purpose: Purpose
       disclosure: Disclosure
-      routine_uses: Routine Uses
+      routine_uses: Routine Uses #should this be in sentence case?
       more_information: For more information
     profile:
       profile_info: Profile information
       profile_info_tt: Why can’t I edit my profile information?
       account_history: Account history
-      login_info: Login information
+      login_info: Account information
```

Comment:
@jeanninehunter Changed for greater consistency in headers. Also because we decided to use "sign in" and not "log in"...and "Sign-in information" just didn't sound right.